### ISSUE TITLE
Enum column in posgresql causes syntax error

### ISSUE THREAD

**Issue by yuku** [k0kubun/sqldef#176]
## --export output

```sql
CREATE TYPE country AS ENUM ('us', 'jp');

CREATE TABLE public.users (
    "id" serial NOT NULL,
    "country" country NOT NULL,
    PRIMARY KEY ("id")
);
```

## Input SQL

```sql
CREATE TYPE country AS ENUM ('us', 'jp');
CREATE TABLE users (
  id SERIAL PRIMARY KEY,
  country country NOT NULL
);
```

## Current output

```
found syntax error when parsing DDL "CREATE TABLE public.users (
    "id" serial NOT NULL,
    "country" country NOT NULL,
    PRIMARY KEY ("id")
)": syntax error at position 76 near 'country'
```

## Expected output

*empty*

**Comment by yuku** [comment]
Since I'm really new to postgresql, please let me know if I'm missing something obvious. 🙏

**Comment by k0kubun** [comment]
The current state seems to be that we only supported parsing the `CREATE TYPE` DDL itself at https://github.com/k0kubun/sqldef/issues/171 and didn't really handle what's defined by that in `CREATE TABLE`, etc. So we'd have to implement that in sqldef and it's probably not your fault.

**Comment by k0kubun** [comment]
Fixed this in v0.11.18.
